src/components/hero.scss

```diff
@@ -95,22 +96,47 @@
         margin-bottom: 35px;
       }
     }
+  }
+}
 
-    .subheaderSubtitleFooterIcon {
-      display: flex;
-      flex-wrap: wrap;
-      justify-content: space-evenly;
+.subheaderSubtitleFooterIcon {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
 
-      ul {
-        display: flex;
-        justify-content: space-around;
-        width: 100%;
-        flex-wrap: wrap;
-        list-style-type: none;
+  ul {
+    display: flex;
+    justify-content: space-around;
+    width: 100%;
+    list-style-type: none;
 
-        li {
-          margin: 0 25px;
-        }
+    li {
+      margin: 0 25px;
+    }
+  }
+}
+
+@media #{$medium-screen} {
+  .subheaderTitleContainer {
+    img {
+      display: none;
+    }
+  }
+
+  .subheaderSubtitleFooter {
+    margin-bottom: 20px;
+  }
+
+  .subheaderSubtitleFooterIcon {
```

Comment:
The icons cannot be aligned to the center, otherwise we can't scroll from the start :) 
<img width="546" alt="image" src="https://user-images.githubusercontent.com/14869087/182247496-b084f137-8a8a-47f9-8bae-19b9d0070791.png">

Reply:
Same thing for the navbar horizontal scrollbar. It cannot be aligned to the center in small devices, otherwise the scroll won't happen :)